docs/building-a-form/common-definitions.md

```diff
@@ -81,63 +86,105 @@ Source: [/src/js/definitions/autosuggest.js](../../src/js/definitions/autosugges
 
 ### Currency
 
-Formats and validates a US currency field. The display includes a leading `$` character. Call this exported function and pass it the label to be used on the field.
+Formats and validates a US currency field that includes a leading `$` character. You can pass this function a label to be used on the field.
```

Comment:
I would change the language in the second sentence to be: "You can pass the `uiSchema` function a label to be used on the field".